src/main/java/org/broadinstitute/hellbender/utils/haplotype/Event.java

```diff
@@ -114,4 +115,11 @@ private static boolean differentLastBase(final Allele ref, final Allele alt) {
         final byte[] altBases = alt.getBases();
         return refBases.length == 0 || altBases.length == 0 || refBases[refBases.length-1] != altBases[altBases.length-1];
     }
+
+    @Override
+    public String toString() {
+        return String.format("[Event @ %s, %s->%s, %s",
```

Comment:
I know its for debugging but the java String.format() call is MUCH slower than string.append() (which string (" " + " " + " ") maps into. These things get called enough in that debug path that I would consider keeping this fast.

Reply:
done